src/js/player.js

```diff
@@ -280,16 +266,24 @@ class Player extends Component {
     // Turn off API access because we're loading a new tech that might load asynchronously
     this.isReady_ = false;
 
-    var techReady = function(){
-      this.player_.triggerReady();
-    };
+    var techReady = Lib.bind(this, function() {
```

Comment:
I think this is duplicating line 297 now? https://github.com/videojs/video.js/pull/2057/files#diff-3b0266ff1c037b289ec624ab25b0272eR297